magellan-library/src/main/java/com/wealthfront/magellan/BaseScreenView.java

```diff
@@ -24,4 +26,7 @@ public final S getScreen() {
     return screen;
   }
 
+  public View inflate(int resource, ViewGroup root) {
+    return inflate(getContext(), resource, root);
```

Comment:
Shouldn't the root be "this"?
Should be the common case, and makes it truly simpler in impl